kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java

```diff
@@ -0,0 +1,196 @@
+package com.applozic.mobicomkit.uiwidgets.conversation.fragment;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.applozic.mobicomkit.uiwidgets.R;
+import com.applozic.mobicommons.commons.core.utils.Utils;
+
+/**
+ * fragment for the feedback input form
+ * @author shubham
+ * @date july 19
+ */
+public class FeedbackInputFragment extends Fragment {
+
+    private static final String TAG = "FeedbackInputFragment";
+
+    EditText editTextFeedback;
+    Button button1;
```

Comment:
Please use proper names which makes it easy to understand what this code is used for

Reply:
changed the names